signac/contrib/project.py

```diff
@@ -260,9 +260,7 @@ def __init__(self, config=None):
         self._lock = RLock()
 
         # Prepare cached properties derived from the project config.
-        self._id = None
-        self._rd = None
-        self._wd = None
+        _invalidate_config_cache(self)
```

Comment:
Is this a deprecation?

Reply:
Ah no sorry this is just making use of an existing API that it probably already should have been. I snuck that in here while I was making these changes but it's not related.